tokio/src/loom/mocked.rs

```diff
@@ -25,6 +25,13 @@ pub(crate) mod sync {
         }
     }
     pub(crate) use loom::sync::*;
+
+    pub(crate) mod atomic {
+        pub(crate) use loom::sync::atomic::*;
+
+        // TODO: implement a loom version
+        pub(crate) type StaticAtomicU64 = std::sync::atomic::AtomicU64;
```

Comment:
This is a bit (ok, a lot)  of a hack, but it is equivalent to before the patch. Unfortunately, right now, loom does not have a way to model static variables.